lib/new_relic/agent/instrumentation/active_record_notifications.rb

```diff
@@ -90,7 +90,8 @@ def log(sql, name = 'SQL', binds = [], type_casted_binds = [], statement_name =
   end
 
   depends_on do
-    !NewRelic::Agent.config[:disable_activerecord_instrumentation] &&
+    !NewRelic::Agent.config[:disable_active_record_instrumentation] &&
+      !NewRelic::Agent.config[:disable_active_record_notifications] &&
```

Comment:
If there a world where someone could have instrumentation without notifications or vice versa?

Reply:
The answer for that world being possible is "definitely". The answer for that possibility potentially causing problems for this change, the answer is "maybe".

Because the agent predates Rails notifications, for some Rails functionality it instruments core Rails libraries and uses the "instrumentation" parameters to toggle that behavior. But for the notification based approaches that we take, we're supposed to use a "notifications" based parameter.

For this particular class it was previously toggled to off only if you used an "instrumentation" parameter, despite it taking a "notifications" approach. I don't want to break that existing functionality, so I left it in place and will now ALSO disable the notifications if the "notifications" parameter is present (because if you want to disable notifications, my thinking is that we should in fact disable notifications).

So there's a chance a situation exists that this change causes a problem for. By retaining backwards compatibility with existing checks and permutations of the related config parameters, I'm hoping I've protected us against potential problems.